src/load_games.py

```diff
@@ -5,14 +5,57 @@
     from src.load_games import get_games
     games = get_games('resources/PGN_database')
 """
-import ast
 import os
 import warnings
 from operator import itemgetter
+import collections
 
+import chess.pgn
 
-def load_games(path: str = 'resources/PGN_database',
-               is_rated: bool = None):
+
+def flatten_list(arg_list):
+    """Flattens a list completely.
+
+    Parameters
+    ----------
+        arg_list : list[list[...]]
+            nested list
+
+    Returns
+    -------
+        list[...]
+            flattened list of games in list.
+
+    """
+    return [item for sublist in arg_list
+            for item in sublist]
+
+
+def flatten_dict(arg_dict):
+    """Flattens a dictionary completely.
+
+        Parameters
+        ----------
+            arg_dict : dict[dict[...]]
+                nested list
+
+        Returns
+        -------
+            dict[...]
+                flattened dict of games in list.
+
+    """
+    items = []
+    for k, v in arg_dict.items():
+        new_key = k
+        if isinstance(v, collections.MutableMapping):
+            items.extend(flatten_dict(v).items())
+        else:
+            items.append((new_key, v))
```

Comment:
What happens (should happen?) when new_key already exists? Current code only uses the last key in the list.